packages/modul-components/src/components/icon/icon.ts

```diff
@@ -28,10 +29,16 @@ export class MIcon extends Vue {
     }
 
     private get spriteId(): string | undefined {
+        const svg: SpritesService = Vue.prototype.$svg;
```

Comment:
Pour être constant avec le reste de modul on devrait utiliser la forme suivate:

`this.$svg`

Reply:
la grosse différence avec le Vue.prototype.$svg c'est qu'on a le autocomplete en typescript

Reply:
Merci. Sera fait dans mon prochain commit.